packages/client-adapters/tests/adapters/logeion/adapter.test.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-env jest */
+/* eslint-disable no-unused-vars */
+import 'whatwg-fetch'
+
+import AlpheiosLogeionAdapter from '@clAdapters/adapters/logeion/adapter'
+
+// import { LogeionFixture } from 'alpheios-fixtures'
+
+describe('logeion/adapter.test.js', () => {
+  console.error = function () {}
+  console.log = function () {}
+  console.warn = function () {}
+
+  beforeEach(() => {
+    jest.spyOn(console, 'error')
+    jest.spyOn(console, 'log')
+    jest.spyOn(console, 'warn')
+  })
+  afterEach(() => {
+    jest.resetModules()
+  })
+  afterAll(() => {
+    jest.clearAllMocks()
+  })
+
+  it('1 AlpheiosLogeionAdapter - constructor uploads config, available', () => {

```

Comment:
I can see the value in keep the integration tests with the live service, but I think this file should then be excluded from the tests that are run by travis.

Reply:
No, there is no real requests to the service.
I execute `getWords` method only inside 5th test, and fetch is mocked there with fixture.
So it is safe. Don't worry!

Reply:
thanks!